src/games/strategy/triplea/delegate/BattleCalculator.java

```diff
@@ -944,171 +933,6 @@ public static CasualtyDetails selectCasualties(final String step, final PlayerID
     return sortedWellEnoughUnitsList;
   }
 
-  private static List<Unit> sortUnitsForCasualtiesWithSupportNew(final Collection<Unit> targetsToPickFrom,
```

Comment:
Hmm. Let me take a look at this. I believe this is the old casualty selector that isn't used anymore.